pytest_splunk_addon/tools/cim_field_report.py

```diff
@@ -0,0 +1,265 @@
+import os.path
+import sys
+import logging
+import json
+import argparse
+import re
+import glob
+import time
+import traceback
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
+
+from pytest_splunk_addon.helmut.manager.jobs import Jobs
+from pytest_splunk_addon.helmut.splunk.cloud import CloudSplunk
+from pytest_splunk_addon.standard_lib.addon_parser import AddonParser
+
+from splunklib import binding
+
+logging.basicConfig(
+    format='%(asctime)s.%(msecs)03d %(name)s %(levelname)s %(message)s', 
+    datefmt='%Y-%m-%d %H:%M:%S',
+    level=logging.ERROR)
+
+LOGGER = logging.getLogger('cim-field-report')
+
+def get_gonfig():
```

Comment:
Should this be `get_config`?

Reply:
yes, fixed

Reply:
also added inline docs

Reply:
Docs tests are failing.

Reply:
It seems fails are not related to the cim fields report code.